android/sdl_android/src/main/java/com/smartdevicelink/transport/SdlBroadcastReceiver.java

```diff
@@ -189,7 +200,7 @@ public void onListObtained(boolean successful) {
 
             if (!didStart) {
                 DebugTool.logInfo(TAG, "attempting to wake up router service");
-                didStart = wakeUpRouterService(context, true, false, device);
+                didStart = wakeUpRouterService(context, true, false, device, vehicleType);
```

Comment:
@kostyaBoss On line 203 you can remove the parameter `vehicleType`.

Reply:
@santhanamk, can't, it should receive the vehicleType, referencing this comment

https://github.com/smartdevicelink/sdl_java_suite/pull/1604#discussion_r563845919